### DISCUSSION TITLE
Resizable columns, Hide Columns, Multi select

### DISCUSSION THREAD

**Discussion by MateenKhan** [community/community#15485] [Other Feature Feedback, Questions, & Ideas]
## **Resizable columns:**
- #### See at a time I will be adding multiple TODO, I would like to resize this column to be big for easy reading

## **Hide columns:**
- #### I am always least interested in what's done, so I just want to hide this column

## **Multi Select:**
- #### I want to select multiple TODO and move then up or down or somewhere below

**Comment by ronnyroeller** [comment]
Another use case for hide column:
We have a column "Icebox" where we store all issues that won't be worked on in the next 3 months. It's an important column (esp for Product Managers) but mostly not relevant for developers on a day-by-day base.
